dashboard/client/dashboard.js

```diff
@@ -5,8 +5,74 @@ Template.dashboard.onCreated(function () {
   // Get reference to template instance
   const templateInstance = this;
 
-  // Create reactive variable for Elasticsearch host
+  // Create reactive variable for Elasticsearch host & data
   templateInstance.elasticsearchHost = new ReactiveVar();
+  templateInstance.elasticsearchData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        // Create data range with interval
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+          },
+          aggs: {
+            // Get average value of response time for each range
+            avg_response_time: {
+              avg: {
```

Comment:
@bryle the citation from article:
> Fundamentally, percentiles are very expensive to calculate. If you want to calculate the 95th percentile, you need to sort all your values from least to greatest, then find the value at myArray[ count(myArray) * 0.95]

And what value of percentiles whould you like to use? In this case, the title "Average response time" will  mislead the users if we are going to use percentile